arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -1586,12 +1699,12 @@
       "x-ms-azure-resource": true,
       "description": "The Resource definition for other than namespace."
     },
-    "NamespaceListResult": {
+    "SBNamespaceListResult": {
```

Comment:
We know that "2015-08-01" version of service bus is ALREADY contains breaking changes. By "already" I mean when we merged this PR https://github.com/Azure/azure-rest-api-specs/pull/1036 it was including breaking changes. We communicated this to service team and requested to ensure they update .net sdk nuget version, @arundsouza is aware of this. But while testing .net SDK generated from this, service team found that compilation is failing due to use of keyword such as "namespace" and type name "Queue" (which introduced due to the removal of postfix 'Resource'). Swagger review team suggested to prefix "SB" to the conflicting types. This PR contains only those changes. @jhendrixMSFT we are good here, we can take this PR.